### ISSUE TITLE
Decreased amyloid beta 42 isoform level 

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7072]
**Preferred term label:**
Decreased amyloid beta 42 isoform level 

**Synonyms**
Decreased beta amyloid 42 level, Decreased Abeta 42 level

**Definition (free text, please give PubMed ID)**
Lower than normal level of amyloid beta 42 found in cerebrospinal fluid.
PMID:31391004

**Parent term (use hpo.jax.org/app)**
Abnormal amyloid beta 42 isoform level

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 104300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
added
